.github/workflows/docker.yml

```diff
@@ -40,4 +41,4 @@ jobs:
           push: true
           build-args: |
             KROXYLICIOUS_VERSION=${{ env.RELEASE_VERSION }} 
-          tags: quay.io/kroxylicious/kroxylicious-developer:${{ env.RELEASE_VERSION }}
+          tags: quay.io/${{ vars.QUAY_ORG }}/kroxylicious-developer:${{ env.RELEASE_VERSION }}
```

Comment:
I needed to factor out QUAY_ORG too as QUAY_USERNAME isn't necessarily the same value (and won't be if using a robot account).